src/mon/mon_types.h

```diff
@@ -473,6 +473,20 @@ namespace ceph {
 	  );
       }
 
+      /**
+       * All the features that require a full quorum to be enabled.
+       *
+       * Features should only be added to this list if you want to
+       * make sure they are only enabled automatically when the quorum
+       * is composed by *all* the monitors in the monmap.
+       */
+      constexpr mon_feature_t get_requires_full_quorum() {
+        return (
+          FEATURE_LUMINOUS |
+          FEATURE_NONE
+          );
+      }
+
```

Comment:
do we need a new category for this?  can't we just apply the full quorum rule for all persistent features?  the fact that they're persistent seems like it implies we want to be careful about enabling them when they can't be disabled...